draft-packages/guidance-block/package.json

```
@@ -33,13 +33,17 @@
   "license": "MIT",
   "dependencies": {
     "@kaizen/component-library": "^12.4.5",
+    "@kaizen/button": "^1.0.4",
     "@kaizen/draft-button": "^5.6.5",
     "@kaizen/draft-illustration": "^3.6.5",
     "@kaizen/draft-tooltip": "^3.7.3",
     "@types/classnames": "^2.3.1",
     "classnames": "^2.3.1",
     "react-media": "^1.10.0"
   },
+  "dependenciesComments": {
+    "@kaizen/draft-button": "The draft dependency should remain here until this is moved out of draft packages"
```

I suggest mentioning the purpose of this is for Elm.

```suggestion
    "@kaizen/draft-button": "The draft dependency should remain here (for Elm) until this is moved out of draft packages"
```

---

I'm going to opt for the latter, short and concise